src/_P112_AS7265x.ino

```diff
@@ -28,6 +28,8 @@
 boolean Plugin_112(byte function, struct EventStruct *event, String& string)
 {
   boolean success = false;
+  String MeasurementStatus;
+  MeasurementStatus = String("Not running");

```

Comment:
Why use a quite resource intensive type like a String?
Also the object will be destructed as soon as you leave this function.
Better to store this status in the `P112_data_struct` object.

Also strings in source code should be wrapped in `F()` wrappers or else they will also use valuable RAM when running.

```c++
 String MeasurementStatus;
  MeasurementStatus = F("Not running");
```

Or even shorter:
```c++
 String MeasurementStatus = F("Not running");
```
But like I said, this is not going to work as you intended it.